cli/src/main.rs

```diff
@@ -145,13 +154,24 @@ fn fetch_metadata(url: &url::Url) -> color_eyre::Result<(String, Vec<u8>)> {
     Ok((hex_data, bytes))
 }
 
-fn codegen<I: Input>(encoded: &mut I) -> color_eyre::Result<()> {
+fn codegen<I: Input>(
+    encoded: &mut I,
+    raw_derives: Vec<String>,
+) -> color_eyre::Result<()> {
     let metadata = <RuntimeMetadataPrefixed as Decode>::decode(encoded)?;
     let generator = subxt_codegen::RuntimeGenerator::new(metadata);
     let item_mod = syn::parse_quote!(
         pub mod api {}
     );
-    let runtime_api = generator.generate_runtime(item_mod, Default::default());
+
+    let mut p: Punctuated<syn::Path, syn::Token![,]> = Punctuated::new();
```

Comment:
This could just be `Vec<syn::Path>` (as far as I can see, that's all it's being used for) and then below, `p.into_iter()` to avoid the `cloned()` :)